### ISSUE TITLE
Typos in About us page on the website

### ISSUE THREAD

**Issue by riya-rjha** [harshalhonde21/EcommerceSpectastyle#176]
### Describe the feature

The About us page consists of text which is looking way too fancy & that can be improvised by choosing a different & subtle font. Plus the alignment of text for all the three different paragraphs are not good. And there are typos such as Delivery is misspelled as Delivary and the starting letter of the line is not from capital. I would want to use TailwindCSS to improvise the UI and make it a whole lot better. Also there is no uniform font size for all the text.

To summarize I would do the following changes:

1. Change of Font
2. Proper alignment
3. Rectifying typos
4. Uniform text size

### Add ScreenShots

![Screenshot (561)](https://github.com/harshalhonde21/EcommerceSpectastyle/assets/112813772/dfa4ad90-104d-4d7b-8dac-f8429ea68b3a)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by harshalhonde21** [comment]
This type of Font is looks great as the website is color full and crazy 

so need for this update

**Comment by riya-rjha** [comment]
@harshalhonde21 I would also request you to take into consideration the font size & the typos in the About Us page. Can I work on that under the GSSOC label, if you please assign it to me?
